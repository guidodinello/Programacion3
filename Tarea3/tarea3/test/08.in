k_disparos_stress 10000 10
Fin