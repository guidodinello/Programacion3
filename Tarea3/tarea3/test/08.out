1>La ganancia maxima con 10 disparo(s) es: 9989000 
2>Fin.
